Dockerfile

```diff
@@ -45,30 +45,13 @@ RUN apk add --no-cache ruby \
   "asciidoctor-pdf:${ASCIIDOCTOR_PDF_VERSION}"
 
 # # # # # # # # # # # # # # # # # # # # # # # # # # # # # #
-# Haskell build for: erd
+# Install erd-go (https://github.com/kaishuu0123/erd-go) as replacement for erd (https://github.com/BurntSushi/erd)
 # # # # # # # # # # # # # # # # # # # # # # # # # # # # # #
-# Alpine 3.16 is the last known working version for building erd (dynamic linking)
-FROM alpine:3.16 AS erd-builder
+FROM golang:1.12-alpine as erd-builder
+RUN apk add --no-cache git \
+    && go install github.com/kaishuu0123/erd-go/v2@v2.0.0
```

Comment:
```suggestion
FROM golang:${ERD_GOLANG_VERSION}-alpine as erd-builder
ARG ERD_VERSION=v2.0.0
## Always use the latest git package
# hadolint ignore=DL3018
RUN apk add --no-cache git \
  && git clone https://github.com/kaishuu0123/erd-go -b "${ERD_VERSION}" /app
WORKDIR /app
RUN CGO_ENABLED=0 GOOS=linux go build
```

- Golang version seems to be 1.15 in the `go.mod` file (gotta track this with updatecli later, once your PR is merged)
- `go install` or `go get` cannot be used the `go.modù` syntax of erd-go is not following the Golang semver properties, leading to errors whatever method is used. This suggestions fixes it by using a go build method to generate the binary instead.


Note: this suggestion requires to add the following lines *before* the first `FROM`:

```
# Golang version defined in https://github.com/kaishuu0123/erd-go/blob/${ERD_VERSION}/go.mod#L3
ARG ERD_GOLANG_VERSION=1.15
```

Reply:
I also adapted the `COPY` command near to the end of the `Dockerfile`.